index.html

```diff
@@ -24,10 +24,37 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
     </li>
   </ul>
 
-  <center><button type="button">Active</button>
-  <button type="button">Completed</button></center>
+  <center>
+  <button v-on:click="showCompleted">Completed</button>
+  <button v-on:click="showActive">Active</button>
 </div>
 
+<ul>
+    <li v-for="todo in activeTodos" :key="todo.id" class="flex">
+      <label>
+        <input type="checkbox" class="nes-checkbox" v-model="todo.done">
+        <span>&nbsp</span>
+      </label>
+      <del v-if="todo.done">{{ todo.text }}</del>
+      <span v-else>{{ todo.text }}</span>
+      <div class="space"></div>
+      <button class="nes-btn is-error padding" v-on:click="removeTodo(todo.id)">X</button>
+    </li>
+  </ul>
+
+  <ul>

```

Comment:
make the unordered list only show up if this.showCompleted is true:
`<ul v-if="showCompleted">`